### ISSUE TITLE
Gitlab time reporting fails because issue.id is used instead of issue.iid

### ISSUE THREAD

**Issue by Blackclaws** [johannesjo/super-productivity#3447]
## Your Environment

- Version used: 10.0.4
- Operating System and version: Archlinux
- Desktop Environment: XFCE

### Expected Behavior

Time tracking works

### Current Behavior

Time tracking does not work but returns 404 because the id instead of the iid is used

### Steps to Reproduce (for bugs)

1. Sync issues from gitlab
2. Work
3. Sync time to gitlab
4. 404 not found

### Can you reproduce this reliably?

Yes

### Console Output

Basically 404 not found

### What is wrong

Essentially the problem is that Superproductivity in the later versions (anything that includes the https://github.com/johannesjo/super-productivity/commit/9577c3e commit) uses the issue id for certain gitlab things instead of the iid, breaking multiple interactions. This should either be reverted or the iid should be used in the correct places.

**Comment by johannesjo** [comment]
Possibly related #3300
